### PR TITLE
New Note  Page UI

### LINKED ISSUE #4: Create new note page UI

### PR DESCRIPTION
**Pull request by saurabh-nikam**
with yup validation schema and constant.js
fixes #4

### CODE REVIEW

File: app/javascript/src/components/Dashboard/Notes/Note.jsx [hunk 1]
```diff
@@ -5,13 +5,13 @@ import { Typography, Tag, Avatar, Tooltip, Dropdown } from "neetoui";
...
 
-const Note = ({ note }) => (
+const Note = ({ note, onDeleteClick }) => (
   <div className="mb-3 box-border w-full border border-gray-300 bg-white p-4 shadow-sm">
     <div className="flex justify-between">
       <Typography style="h4">{note.title}</Typography>
       <Dropdown buttonStyle="text" icon={MenuVertical} position="bottom-end">
         <li>Edit</li>
-        <li>Delete</li>
+        <li onClick={() => onDeleteClick(note.id)}>Delete</li>
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/Note.jsx]: https://bigbinary.slack.com/archives/C01T5Q4DZAR/p1663147237774019?thread_ts=1663060378.971029&cid=C01T5Q4DZAR

File: app/javascript/src/components/Dashboard/Notes/constants.js [hunk 2]
```diff
@@ -34,7 +34,8 @@ export const NOTES_FORM_VALIDATION_SCHEMA = yup.object().shape({
         value: yup.string().required(),
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/constants.js]: Go through the Slack thread once again. I have added a point regarding this validation and how to enhance it.

File: app/javascript/src/components/Dashboard/Notes/DeleteAlert.jsx [hunk 3]
```diff
@@ -1,30 +1,15 @@
...
-};
+const DeleteAlert = ({ onClose, handleDelete }) => (
+  <Alert
+    isOpen
+    message="Are you sure you want to continue? This cannot be undone."
+    title="Delete note?"
+    onClose={onClose}
+    onSubmit={handleDelete}
+  />
+);
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/DeleteAlert.jsx]: Go through the reviews I had pointed out via Github in one of your other PRs and make the change.

**Review by yedhink — CHANGES_REQUESTED** [review]
@saurabh-nikam

File: app/javascript/src/components/Dashboard/Notes/DeleteAlert.jsx (hunk 3 above)
Comment (saurabh-nikam) [reply app/javascript/src/components/Dashboard/Notes/DeleteAlert.jsx]: Changed in previous PR

Comment (yedhink) [reply app/javascript/src/components/Dashboard/Notes/DeleteAlert.jsx]: The message you had added was too verbose. Keep it compact yet on point:

```
Are you sure you want to delete the note "${note.name}"? This action cannot be undone."
```

Comment (saurabh-nikam) [reply app/javascript/src/components/Dashboard/Notes/DeleteAlert.jsx]: ok, sure this one is on point and compact as well.

File: app/javascript/src/components/Dashboard/Notes/Note.jsx (hunk 1 above)
Comment (saurabh-nikam) [reply app/javascript/src/components/Dashboard/Notes/Note.jsx]: Replaced `onDeleteClick` with `handleDeleteButton`

File: app/javascript/src/components/Dashboard/Notes/constants.js (hunk 2 above)
Comment (saurabh-nikam) [reply app/javascript/src/components/Dashboard/Notes/constants.js]: Done

**Pull request merged**